src-electron/util/zcl-util.js

```diff
@@ -75,6 +75,23 @@ function commandComparator(a, b) {
   return 0
 }
 
+/**
+ * Comparator for sorting events.
+ *
+ * @param {*} a
+ * @param {*} b
+ * @returns -1, 0 or 1
+ */
+function eventComparator(a, b) {
+  if (a.manufacturerCode < b.manufacturerCode) return -1
+  if (a.manufacturerCode > b.manufacturerCode) return 1
+
+  if (a.hexCode < b.hexCode) return -1
+  if (a.hexCode > b.hexCode) return 1
```

Comment:
So here, sort by code, not hexcode, once we switch to the existing shared event mapping?

Reply:
done